src/components/scenes/StakingChangeScene.js

```diff
@@ -205,15 +205,18 @@ export const StakingChangeSceneComponent = (props: Props) => {
   }, [amount, currencyPlugin])
 
   useEffect(() => {
-    if (!selectedFioAddress) {
-      const fioAddress = fioAddresses.find(({ walletId: fioAddressWalletId }) => fioAddressWalletId === walletId)
+    if (!selectedFioAddress && fioAddresses != null && fioAddresses.length > 0) {
```

Comment:
`fioAddresses` should never be nullish according to the prop's type. We can remove the null-check, and if you're concerned that the static types don't give use guarantees about it not being nullish, use `?.length`.